### PR TITLE
add basic structure

### PR DESCRIPTION
**Pull request by justOleh**

### CODE REVIEW

File: src/post_processors/AbstractPostProcessor.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
+from abc import ABC, abstractmethod
```
Comment (Dmytro-Shvetsov) [inline src/post_processors/AbstractPostProcessor.py]: Refactor file name pls

File: src/sorters/AbstractSorter.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
+from abc import ABC, abstractmethod
```
Comment (Dmytro-Shvetsov) [inline src/sorters/AbstractSorter.py]: Refactor file name pls

File: src/sorters/Categorizer.py [hunk 3]
```diff
@@ -0,0 +1,9 @@
+from src.sorters.AbstractSorter import AbstractSorter
```
Comment (Dmytro-Shvetsov) [inline src/sorters/Categorizer.py]: Refactor file name pls

File: src/post_processors/AbstractPostProcessor.py (hunk 1 above)
Comment (justOleh) [reply src/post_processors/AbstractPostProcessor.py]: What would you like it be called?

Comment (Dmytro-Shvetsov) [reply src/post_processors/AbstractPostProcessor.py]: abstract_post_processor.py

Comment (justOleh) [reply src/post_processors/AbstractPostProcessor.py]: Maybe we can skip PostProcessor part in file name

Comment (Dmytro-Shvetsov) [reply src/post_processors/AbstractPostProcessor.py]: Sure, I usually name such modules base.py

Comment (justOleh) [reply src/post_processors/AbstractPostProcessor.py]: Okay, I think we can avoid even underscores, so let call it just src/post_processors/abstract.py

**Review by Dmytro-Shvetsov — APPROVED** [review]

**Pull request merged**